tests/app/integrations/comp_and_pen/test_scheduled_message_helpers.py

```diff
@@ -0,0 +1,201 @@
+from decimal import Decimal
+
+import pytest
+
+from app.integrations.comp_and_pen.scheduled_message_helpers import CompPenMsgHelper
+from app.models import SMS_TYPE
+from app.va.identifier import IdentifierType
+
+
+@pytest.fixture
+def msg_helper(mocker, dynamodb_mock) -> CompPenMsgHelper:
+    # Mocks necessary for dynamodb
+    mocker.patch('boto3.resource')
+    helper = CompPenMsgHelper('test')
+    mocker.patch.object(helper, 'dynamodb_table', dynamodb_mock)
+    return helper
+
+
+def test_ut_get_dynamodb_comp_pen_messages_with_empty_table(msg_helper):
+    # Invoke the function with the mocked table and application
+    messages = msg_helper.get_dynamodb_comp_pen_messages(message_limit=1)
+
+    assert messages == [], 'Expected no messages from an empty table'
+
+
+def test_get_dynamodb_comp_pen_messages_filters(msg_helper, sample_dynamodb_insert):
+    """
+    Items should not be returned if any of these apply:
+        1) payment_id equals -1
```

Comment:
Should we update these test that are still using a payment_id == -1 logic? 

We're no longer creating defaults in the DynamoDB table. This might be out of scope for the ticket.

Reply:
updated